src/extension/kernel.ts

```diff
@@ -157,16 +157,22 @@ export class Kernel implements Disposable {
 
     this.messaging.postMessage({ from: 'kernel' })
     this.panelManager = new PanelManager(context)
-    this.#disposables.push(
-      this.messaging.onDidReceiveMessage(this.#handleRendererMessage.bind(this)),
-      workspace.onDidOpenNotebookDocument(this.#handleOpenNotebook.bind(this)),
-      workspace.onDidSaveNotebookDocument(this.#handleSaveNotebook.bind(this)),
-      window.onDidChangeActiveColorTheme(this.#handleActiveColorThemeMessage.bind(this)),
-      window.onDidChangeActiveNotebookEditor(this.#handleActiveNotebook.bind(this)),
-      this.panelManager,
-      this.onVarsChangeEvent,
-      this.registerTerminalProfile(),
-    )
+
+    // DO NOT COMMIT this was a hack because I was getting crashes when trying to run the extension
```

Comment:
let's figure out what's going on - sooner or later the lack of running these handlers will rear it's ugly head

Reply:
Seems to be working for me now. I have no idea what changed. I uninstalled the extension and reinstalled it as part of trying to install a dev version of the extension in my vscode; so maybe it was that 🤷